utils/utils.go

```diff
@@ -94,6 +94,13 @@ func SanitizeHexString(hex string) string {
 	return hex
 }
 
+func MaxBigInt(a, b *big.Int) *big.Int {
```

Comment:
This can be removed now I think. If we want to keep it for future use, lets have unit tests for it.